purrsoft/src/store/api/index.ts

```diff
@@ -0,0 +1,26 @@
+import { SerializedError } from '@reduxjs/toolkit';
+import { FetchBaseQueryError } from '@reduxjs/toolkit/query';
+export { endpoints as authEndpoints } from './auth';
```

Comment:
authEndpoints nu e folosit nicaieri in cod. Oare trebuie pe mai tarziu sau poate fi sters?